sorc/prepobs_prepdata.fd/w3xtovseds.f

```diff
@@ -217,8 +217,8 @@ SUBROUTINE W3XTOVSEDS(IUNIT,IBDATE,IBUFTN,ISATOB,PBOT,DSNAME,
       REAL GEOOPR(18),TLAYER(15),PLAYER(16),RINC(5)
 
       INTEGER IBUFTN(140),ISATOB(3),JDATE(5),JDUMP(5),IBIT(2)
-
-      INTEGER(8) IDSDMP_8,JDUMP_8(5),KDATE(8),LDATE(8)
+C     DEFINED IDSAT as INTEGER(8) to accomodate WCOSS2 compiler
```

Comment:
Put this comment up above around line 70 in the official documentation timeline. Follow the formatting already in place.